EIPS/eip-7425.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: 7425
+title: Tokenized Reserve
+description: Reserve with user auditing
+author: Jimmy Debe (@jimstir)
+discussions-to: https://ethereum-magicians.org/t/eip-7425-tokenized-reserve/15297
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-30
+requires: 20, 4626
+---
+
+## Abstract
+
+A proposal for a tokenized reserve mechanism. The reserve allows an audit of on-chain actions of the owner. Using [ERC-4626](../EIPS/eip-4626.md), stakeholders can create shares to show support for actions in the reserve.
+
+## Motivation
+
+Tokenized reserves are an extension of tokenized vaults. The goal is to create a reserve similar to a real world reserve an entity has as a backup incase regular funds run low. In the real world, an entity will have certain criteria to access reserve funds. In a decentralized environment, an entity can incorporate stakeholders into their criteria. Hopefully this will help entities who participate in decentralized environments to be transparent with stakeholders.
```

Comment:
```suggestion
Tokenized reserves are an extension of tokenized vaults. The goal is to create a reserve similar to a real world reserve an entity has as a backup in case regular funds run low. In the real world, an entity will have certain criteria to access reserve funds. In a decentralized environment, an entity can incorporate stakeholders into their criteria. This will help entities who participate in decentralized environments to be transparent with stakeholders.
```